IMC_WebDev/#Driving_the_car_and_flowing_the_water_with_CSS/proposal.md

```diff
@@ -0,0 +1,30 @@
+**[Step-By-Step Technical Blog Guide](https://hq.bitproject.org/how-to-write-a-technical-blog/)**
+
+### 📌 Step 1
+
+**TITLE:**
```

Comment:
The title is a little too specific. Try something more general, like "An Introduction to Web Animations using CSS".